MetaMask/metamask-extension
# Add Skylink support to ENS resolver

Explanation:  
This PR extends the current ENS to IPFS resolver with the capability of resolving content hashes to skylinks. ENS added Skynet has been supported by ENS for a while now (cfr. https://github.com/ensdomains/content-hash/pull/7) and we would like ENS domains to be able to resolve to skylinks through MetaMask's listener.

Example: 
https://siasky.net/fAD570XqL8xQGJ3bxlS7RncsGes7ViGxcR5zYf5WXIuRNw

Manual testing steps:  
  - build the project
  - load extension in FireFox
  - type `skynetlabs.eth/` and hit enter in the address bar
  - verify it redirects to https://20062i9hn2944pcu789tdidf71i1s34hl1t2c4ncq7stdbc5i11a888.siasky.net/
